www/front_src/src/Header/SwitchThemeMode /index.tsx

```diff
@@ -96,19 +97,20 @@ const SwitchThemeMode = (): JSX.Element => {
   const switchEndPoint = './api/latest/configuration/users/current/parameters';
 
   const switchThemeMode = (): void => {
+    const theme = isDarkMode ? ThemeMode.light : ThemeMode.dark;
+    if (!pathname.includes('php')) {
```

Comment:
extract this check to a variable (named isCurrentPageLegacy) to reuse it and make it even clearer